lib/async_cache/workers/sidekiq.rb

```diff
@@ -7,8 +7,19 @@ class SidekiqWorker
       include Base
       include Sidekiq::Worker
 
-      # Only allow one job per set of arguments to ever be in the queue
-      sidekiq_options :unique => :until_executed
+      # Pulled out into a module so it can be tested.
+      module Options
+        def self.included(mod)
+          if defined?(Sidekiq::Enterprise)
+            mod.sidekiq_options unique_for: 10.minutes
```

Comment:
These `10.minutes` seems quite arbitrary. Wouldn't it be better to have it living in a config somewhere?